src/itree.cpp

```diff
@@ -16,13 +16,20 @@
 #include <regex>
 #include <stdexcept>
 #include <string>
+#include <variant>
+#include <cassert>
+#include <fnmatch.h>
```

Comment:
Let's add an empty line above this include since it isn't part of the C++ std lib.

Reply:
done